Tasks/RunPowerShellOnTargetMachines/task.json

```diff
@@ -1,27 +1,29 @@
 ﻿{
     "id": "3B5693D4-5777-4FEE-862A-BD2B7A374C68",
-    "name": "DeployToTfsEnvironment",
-    "friendlyName": "Deploy To TFS Environment",
-    "description": "Deploy to TFS Environment.  Executes PowerShell commands remotely on machine(s). This task is under Preview, please contact devops_tools@microsoft.com to enable this task fully on the backend for your account. Please refer http://aka.ms/ci_cd for additional guidance.",
+    "name": "RunPowerShellOnTargetMachines",
+    "friendlyName": "Run PowerShell on Target Machines",
+    "description": "Run PowerShell on Target Machines.  Executes PowerShell commands remotely on machine(s). This task is under Preview, please contact devops_tools@microsoft.com to enable this task fully on the backend for your account. Please refer http://aka.ms/ci_cd for additional guidance.",
     "category": "Deploy",
     "visibility": [
                 "Preview",
-                "Build"
+                "Build",
+                "Release"
```

Comment:
i think when we have both build and RM. we just need to specify Preview. only for dta task we need preview and build